service/controller/app/v1/resource/status/create.go

```diff
@@ -30,7 +30,10 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 
 	chart, err := cc.G8sClient.ApplicationV1alpha1().Charts(r.watchNamespace).Get(name, metav1.GetOptions{})
 	if errors.IsNotFound(err) {
-		return microerror.Maskf(notFoundError, "chart %#q in namespace %#q", name, r.watchNamespace)
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("did not find chart %#q in namespace %#q", name, r.watchNamespace))
+		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource")
+		return nil
```

Comment:
During testing I hit timing issues because the chart CR had not been created yet. We should just cancel the resource and try again later when the chart is not found.